### DISCUSSION TITLE
How to set PDP message to multicast only

### DISCUSSION THREAD

**Discussion by advancenXX** [eProsima/Fast-DDS#5105] [Q&A]
I am using fastdds 2.7.0, and find dds endpoint will send PDP message through unicast after discovery others.
How to restrict PDP message to multicast?

**Comment by paxifaer** [comment]
After PDP discovery, EDP is used to discover individual data endpoints. What you see is the EDP data message. The DDS protocol clearly stipulates that simple service discovery EDP uses unicast.

**Comment by JesusPoderoso** [comment]
Hi @advancenXX, thanks for using Fast DDS.
As @paxifaer pointed out, [PDP discovery phase](https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery.html#discovery-phases) is performed by default through multicast, and once the participants have discovered each other, the [EDP discovery phase](https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery.html#discovery-phases) is performed through unicast.
